src/app/db/tables.py

```diff
@@ -28,7 +28,7 @@
     "accesses",
     metadata,
     Column("id", Integer, primary_key=True),
-    Column("login", String(50)),
+    Column("login", String(50), index=True, unique=True),  # index for fast lookup
```

Comment:
What is this doing? (sorry I might have not gone through the whole documentation :sweat_smile: )

Reply:
An index allows the database to fetch rows faster when filter clause contains the index column -> accesses table is always queried using the login. See: [postgresql documentation](https://www.postgresql.org/docs/12/indexes-intro.html)